sys/riotboot/flashwrite.c

```diff
@@ -150,6 +151,34 @@ int riotboot_flashwrite_putbytes(riotboot_flashwrite_t *state,
     return 0;
 }
 
+int riotboot_flashwrite_invalidate(int slot)
+{
+    extern const unsigned riotboot_slot_numof;
+
+    if (riotboot_slot_numof == 1){
+        LOG_WARNING(LOG_PREFIX "Only one slot configured\n");
+        return -1;
+    }
```

Comment:
This behavior does not match the function documentation, @bergzand do you think it should allow invalidating in this case?

Reply:
This check is done to make sure the device still has at least one firmware image to be running after invalidation, so a reboot after calling the API won't make the device with no image to run. And thus the warning generated to not allow the invalidation in this case.

Reply:
I think this is fine, but the API documentation should reflect that behavior.